src/AppInstallerCLICore/Workflows/DependencyNodeProcessor.cpp

```diff
@@ -41,7 +41,18 @@ namespace AppInstaller::CLI::Workflow
         const auto& package = match.Package;
         auto packageId = package->GetProperty(PackageProperty::Id);
         m_nodePackageInstalledVersion = package->GetInstalledVersion();
-        m_nodePackageLatestVersion = package->GetLatestAvailableVersion();
+
+        PinBehavior pinBehavior;
```

Comment:
The compiler doesn't complain because it can tell that it is set in both branches of the conditional. In my mind it is clearer this way. If I initialize it, in my mind it opens the questions of "when does it stay with this value?", "why is this the default?", neither of which really matters.

I just checked and if it can stay uninitialized, the compiler errors out, so there's no risk of using it uninitialized even if I don't do it here.